src/components/internshipCard/InternshipCard.jsx

```diff
@@ -25,17 +31,21 @@ export const InternshipCard = ({
     />
     <CardContent className="cardContent">
       <Typography color="primary" gutterBottom variant="h6" component="div">
-        {getFieldLabel(title)}
+        {name}
       </Typography>
       <Typography variant="subtitle2" color="text">
-        {getFieldLabel(dateInterval)}
+        {dayjs(registrationStartDate, registrationFinishDate).format(
+          'D MMMM YYYY - D MMMM YYYY',
+        )}
       </Typography>
       <Typography variant="subtitle2" color="text">
-        {getFieldLabel(numberOfMembers)}
+        {getFieldLabel('internships.program.selection')}
+        {maxCandidateCount}
+        {getFieldLabel('internships.program.members')}
       </Typography>
     </CardContent>
     <Box display="flex" flexDirection="row" justifyContent="center">
-      <Button className="internshipCardButton" size="small">
+      <Button raised className="internshipCardButton" size="small">
```

Comment:
are you sure the button needs this prop? 
![image](https://user-images.githubusercontent.com/55104858/140826835-8cba40b1-4284-456c-842f-00feae308e3c.png)

Reply:
you are right, no need, forgot to delete after changes in sass file